src/app/complex/complex-results/complex-navigator-buttons/complex-navigator-buttons.component.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+
+@Component({
+  selector: 'cp-complex-navigator-buttons',
+  templateUrl: './complex-navigator-buttons.component.html',
+  styleUrls: ['./complex-navigator-buttons.component.css']
+})
+export class ComplexNavigatorButtonsComponent {
+  @Output() interactorsSortingChange = new EventEmitter<string>();
+  @Output() organismIconDisplayChange = new EventEmitter<boolean>();
+  @Output() interactorTypeDisplayChange = new EventEmitter<boolean>();
+  @Output() IDDisplayChange = new EventEmitter<boolean>();
+
+  @Input() interactorsSorting: string;
+  @Input() organismIconDisplay: boolean;
+  @Input() interactorTypeDisplay: boolean;
+  @Input() IDDisplay: boolean;
+
+  typeOfDisplay: string;
+
+  InteractorsSorting(typeOfSorting: string) {
+    this.interactorsSortingChange.emit(typeOfSorting);
+    switch (this.typeOfDisplay) {
+      case 'compact':
+        this.compactDisplay();
+        break;
+      default:
+        this.detailedDisplay();
+        if (typeOfSorting === 'Type') {
+          this.interactorTypeDisplay = false;
+        } else if (typeOfSorting === 'Organism') {
+          this.organismIconDisplay = false;
+        }
+    }
+    this.changesEmitter();
+  }
+
+  DisplayingOrganism() {
+    this.organismIconDisplay = !this.organismIconDisplay;
+    this.changesEmitter();
```

Comment:
If `organismIconDisplay` is set to true, we need to set detailed display to true too. It may already be true, but it doesn't hurt setting it anyway.

```
    this.organismIconDisplay = !this.organismIconDisplay;
    if (this.organismIconDisplay) {
        this.IDDisplay = true;
    }
    this.changesEmitter();
```

Same change needed in `DisplayingType` and `DisplayingID` with the relevant variables.

This is to fix the issue of, being in compact display and clicking on one of the detailed checkboxes.